applications/jupyter-lab-hpc-native/tapisjob_app.sh

```diff
@@ -0,0 +1,158 @@
+#!/bin/bash
+
+echo "TACC: job ${SLURM_JOB_ID} execution at: $(date)"
+
+# TAP Port
+LOCAL_PORT=5902
+
+mkdir -p $PWD/Jupyter
+cd Jupyter
+
+# This file will be located in the directory mounted by the job.
+SESSION_FILE="delete_me_to_end_session"
+touch $SESSION_FILE
```

Comment:
Let's do away with session files, and can instead run the app directly and let the job hang there on that process:
https://github.com/TACC/WMA-Tapis-Templates/blob/main/applications/jupyter-lab-hpc/run.sh#L47

Reply:
I replaced it with:

# Launch jupyter-lab and hang the job on this process directly
jupyter-lab ${JUPYTER_ARGS}